src/WebJobs.Script/ManagedDependencies/DependentLibrary.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script.ManagedDependencies
+{
+    /// <summary>
+    /// Represents dependent library mentioned in the host.json file
+    /// </summary>
+    internal class DependentLibrary
+    {
+        /// <summary>
+        /// Gets or sets the module name in host.json file
```

Comment:
The configuration source is abstracted away from the runtime, so it would be good to avoid documentation that specifies this is coming from `host.json` as it may not be accurate.